packages/core/src/components/callout/_callout.scss

```diff
@@ -87,6 +87,14 @@ $callout-padding-compact: $pt-grid-size;
     }
   }
 
+  &.#{$ns}-minimal,
+  &.#{$ns}-minimal.#{$ns}-intent-primary,
+  &.#{$ns}-minimal.#{$ns}-intent-success,
+  &.#{$ns}-minimal.#{$ns}-intent-warning,
+  &.#{$ns}-minimal.#{$ns}-intent-danger {
+    background-color: transparent;
+  }
```

Comment:
Instead of explicitly overwriting styles, could we instead use a negation pseudo-class to prevent the background color from being applied in the first case if the callout is minimal? e.g.

```scss
.#{$ns}-callout {
  &:not(.#{$ns}-minimal) {
    background-color: rgba($gray3, 0.15);
  }

  .#{$ns}-dark & {
    &:not(.#{$ns}-minimal) {
      background-color: rgba($gray3, 0.2);
    }
  }

  @each $intent, $color in $pt-intent-colors {
    &.#{$ns}-intent-#{$intent} {
      &:not(.#{$ns}-minimal) {
        background-color: rgba($color, 0.1);
      }

      .#{$ns}-dark & {
        &:not(.#{$ns}-minimal) {
          background-color: rgba($color, 0.2);
        }
      }
    }
  }
}
```